addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -661,15 +780,51 @@ export class GraphRenderer extends Component {
             plugins: {
                 legend: this.getLegendOptions(),
                 tooltip: this.getTooltipOptions(),
+                gridOnTop: mode != "line" ? false : true,
             },
             elements: this.getElementOptions(),
-        };
+            onResize: () => {this.resizeChart(options)},
+            animation: this.getAnimationOptions(),
+        }
         if (!disableLinking && mode !== "line") {
             options.onClick = this.onGraphClicked.bind(this);
         }
+        if(mode === "line") {
+            options.interaction = {
+                mode: 'index',
+                intersect: false,
+            }
+            options.plugins.gridOnTop = {
```

Comment:
no need for this if you refactor the way to set your plugin gridOnTop.

Reply:
Are you sure @Polymorphe57 ? How can I set my parameters for the line chart if not here?
Or do you want to skip the custom parameters and set the correct value directly in the "gridOnTop" function? 🤔

Reply:
I think that in the config object, you could have
plugins: [gridOnTop]
If yes, you could add that in getChartConfig only for line chart and have your plugin more specialized. Am I wrong?

Reply:
Yes indeed, thanks ! 👍